vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -314,45 +314,6 @@ - (void)onRecoverWallet:(ledger::Result)result balance:(double)balance grants:(s
   }
 }
 
-- (NSString *)BATAddress
```

Comment:
Since you deleted these, you should delete their accompanying `@property` decelerations in BATBraveLedger.h, located here:

https://github.com/brave/brave-core/blob/f42cb77281d8a9348803fe4163091cf95e06eb50/vendor/brave-ios/Ledger/BATBraveLedger.h#L86-L89

Reply:
removed